headers/pmdsky.h

```diff
@@ -1,6 +1,13 @@
 #ifndef HEADERS_PMDSKY_H_
 #define HEADERS_PMDSKY_H_
 
+#include "versions.h"
+
+// Default to NA if there's no predefined version
+#ifndef PMDSKY_VERSION
+#define PMDSKY_VERSION PMDSKY_NA
```

Comment:
What are the implications of choosing one default version or another? Does it matter for anything?

Reply:
I think the only implications should be small, and the choice is more-or-less arbitrary (NA has been my "default" for this repo for various things, and NA/EU are mostly the same right now besides the two script enums):
- someone wasn't reading the setup instructions properly and imports `pmdsky.h` instead of `pmdsky_*.h`; at least this way the imported headers are always valid by some measure
- if you don't care about the version-specific stuff and are lazy or the code is older; one of my other projects imports `pmdsky.h` and I don't feel super inclined to change it

Ideally anyone using the headers for something sensitive to version (Ghidra, c-of-time, etc.) would always use the appropriate version-aware headers.